doc/proposals/sdk-code-annotations.md

```diff
@@ -2,7 +2,10 @@
 
 Implementation Owner: @estroz
 
-Status: Done
+> Status: **completed**
+> 
+> This proposal has been completed.
```

Comment:
Do we have docs for these? If so, we should add a link

/cc @estroz

Reply:
They'll be added in #1162. Once this is merged I can update this section.

Reply:
It was added here. Really tks 👍